models/example/order_calculations.sql

```diff
@@ -0,0 +1,29 @@
+with orders as (
+
+    select * from {{ ref('stg_orders') }}
+
+),
+
+orderitems as (
+    
+    select * from {{ ref('stg_order_items') }}
+),
+
+calc as (
+
+    select 
+        orders.customer_id, 
```

Comment:
this is another situation where you want to `select *, add_new_fields` so that your downstream model has all of the fields and more for you to select from

we can talk about this in more detail during our check in tomorrow!